formats/statistical_data_set/publisher/details.json

```diff
@@ -0,0 +1,63 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+   "required": [
+     "body",
+     "first_public_at",
+     "government",
+     "political"
+   ],
+  "properties": {
+     "body": {
+       "type": "string"
+     },
+     "first_public_at": {
+       "type": "string",
+       "format": "date-time"
+     },
+     "change_history": {
+       "$ref": "#/definitions/change_history"
+     },
+     "tags": {
+       "type": "object",
+       "additionalProperties": false,
+       "properties": {
+         "browse_pages": {
+           "type": "array",
+           "items": {
+             "type": "string"
+           }
+         },
+         "topics": {
+           "type": "array",
+           "items": {
+             "type": "string"
+           }
+         },
+         "policies": {
+           "type": "array",
+           "items": {
+             "type": "string"
+           }
+         }
+       }
+     },
+     "government": {
+       "$ref": "#/definitions/government"
+     },
+     "political": {
+       "$ref": "#/definitions/political"
+     },
+     "emphasised_organisations": {
```

Comment:
[Trello ticket](https://trello.com/c/g2zCKSFQ/520-refactor-emphasised-organisations-and-tags-in-details-json-in-govuk-content-schemas) created.